src/modules/layout/__tests__/HeaderDropDown.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import HeaderDropDown from '../components/HeaderDropDown'
+
+describe('HeaderDropDown', () => {
+  const MockNode = () => <div>Here comes the DropDown</div>
+  let wrapperComponent
+
+  beforeEach(() => {
+    wrapperComponent = shallow(<HeaderDropDown iconSrc='/someImg'>
+      <MockNode />
+    </HeaderDropDown>)
+  })
+
+  test('should be wrapped by onClickOutside', () => {
+    // todo: ??
```

Comment:
_Don't forget me :)_

Reply:
Do you have any idea on how to test that?

Reply:
you can use find() to look for a HOC with a specific displayName. Like `find('locationFetcher')`

Reply:
Hmm... this seems so buggy... I've worked on that for an hour now and it just won't work. The find(...) returns an empty string and when I call `wrappedComponent.name()` I get `HeaderDropDown` instead of `OnClickOutside(HeaderDropDown)` what I would expect.
I would skip that test now, since it's partially covered by the 'should match snapshot' test, because we need to dive there^^